lib/actions/build.js

```diff
@@ -152,8 +153,13 @@ Examples:
 		}
 		delete params.source;
 
-		options.convertEol = options['convert-eol'] || false;
-		delete options['convert-eol'];
+		if (isV12()) {
+			options.convertEol = options['convert-eol'] || false;
+			delete options['convert-eol'];
+		} else {
```

Comment:
Should this `if-else` be the other way around? Swapping the `if` block with the `else` block.

Reply:
Yep, that's done already but not yet pushed.